hrjobcontract/js/src/job-contract/controllers/contract.js

```diff
@@ -69,25 +69,17 @@ define([
             };
 
             function updateContractList(newEndDate){
```

Comment:
I think you understand now what this function is for and when it is used, please write the documentation for it
```js
/**
 * Lorem ipsum
 *
 * @param {type} newEndDate description
 */
function updateContractList(newEndDate) {